sig-node/CONTRIBUTING.md

```diff
@@ -2,24 +2,98 @@
 
 Welcome!
 
-## For Kubernetes Contributions
+Thank you for your interest in contributing to SIG Node. SIG Node is one of the biggest SIGs in Kubernetes.
+Reliability and performance of millions of nodes running critical applications in production rely on the quality of your contribution(s).
+The diversity of workloads, environments, and the scale SIG Node needs to support makes every code change risky as all the side effects need to be evaluated.
+And the contribution can realistically only be evaluated by a small set of maintainers.
+
+Please make sure you understand and are up to the challenge. The contributing instructions are designed to help you.
+
+## For Kubernetes Code Contributions
 
 Read the [Kubernetes Contributor Guide](https://github.com/kubernetes/community/tree/master/contributors/guide#contributor-guide).
 
 If you aspire to grow scope in the SIG, please review the [SIG Node contributor ladder](./sig-node-contributor-ladder.md) for SIG specific guidance.
 
 ### For Enhancements
 
-SIG Node enhancements are available in the <https://github.com/kubernetes/enhancements/tree/master/keps/sig-node>.
+Find out if [your thing is an enhancement a.k.a KEP](https://github.com/kubernetes/enhancements/?tab=readme-ov-file#is-my-thing-an-enhancement).
+A good way to do that is to open and issue and get feedback from node reviewers and approvers. You can even come present your idea at a weekly sig-node meeting.
+
+If you plan to contribute an enhancement, please prepare yourself for at least 1 year of engagement.
+A KEP takes at least 3 kubernetes releases to move from alpha to beta to GA. If there are API / kubelet skew considerations, it may take even longer. 
+SIG Node expects contributors to commit to land a KEP to GA stage to avoid [permanent betas](https://kubernetes.io/blog/2020/08/21/moving-forward-from-beta/#avoiding-permanent-beta).
```

Comment:
WDYT of 

`SIG Node expects that contributors commit to taking a KEP to GA stage to avoid`